src/submission/validation-clinical/therapy.ts

```diff
@@ -45,14 +45,101 @@ export const validate = async (
     throw new Error("Can't call this function without a registerd donor & therapy record");
   }
 
-  const errors: SubmissionValidationError[] = [];
+  let errors: SubmissionValidationError[] = [];
   const treatment = getTreatment(therapyRecord, mergedDonor, errors);
   if (!treatment) return { errors };
-  checkTreatementHasCorrectTypeForTherapy(therapyRecord, treatment, errors);
+  // TODO: Refactor use of push
+  checkTreatmentHasCorrectTypeForTherapy(therapyRecord, treatment, errors);
+
+  const {
+    clinicalInfo: { treatment_type },
+    therapies,
+  } = treatment;
+
+  const isRadiationTreatment =
+    Array.isArray(treatment_type) &&
+    treatment_type?.includes('Radiation therapy') &&
```

Comment:
`treatment_type` will always have `includes` property because its core js array api and the previous line makes sure its an array

Reply:
i misread this. this is checking if `treatment_type` is defined? which previous line should cover

Reply:
True, updated